lib/version.js

```diff
@@ -0,0 +1,115 @@
+var RUNTIME_PACKAGE = require('../package.json'),
+
+    /**
+     * List of postman packages to include while generating dependency object
+     *
+     * @const
+     * @type {String[]}
+     */
+    PACKAGES_TO_CONSIDER = [
+        'chai-postman',
+        'postman-collection',
+        'postman-request',
+        'postman-sandbox',
+        'uniscope',
+        'uvm'
+    ],
+
+    /**
+     * Generate object containing version and dependencies of Runtime or given module.
+     *
+     * @param {String} [moduleName] - Module name
+     * @param {Boolean} [deepDependencies=true] - Do depth dependencies traversal or stop at root
+     * @param {Object} [moduleData={}] - Object to store module data
+     * @returns {Object}
+     *
+     * @example <caption>Returned Object Structure</caption>
+     * var runtime = require('postman-runtime');
+     * runtime.version();
+     * {
+     *     version: '7.6.1',
+     *     dependencies: {
+     *         'postman-collection' : {
+     *             version: '3.4.2',
+     *             dependencies: {
+     *                 'postman-request': {
+     *                     version: '2.88.1-postman.5'
+     *                 }
+     *             }
+     *         },
+     *         'postman-request': {
+     *             version: '2.88.1-postman.5'
+     *         }
+     *     }
+     * }
+     */
+    getVersionData = function (moduleName, deepDependencies, moduleData) {
+        !moduleData && (moduleData = {});
+        (deepDependencies === undefined) && (deepDependencies = true);
```

Comment:
If you are referring to the function getVersionData, , it generates and returns the dependency object as given in the comment above.

Or if you are referring to the first two lines of the function, those are to set the default values of function arguments. 

- The argument moduleData will contain an object with version of dependency given in the moduleName argument(if function is called recursively, empty object otherwise).

- The argument deepDependencies is a boolean (true by default) indecating weather to include recursive dependency in generated object or not.

Reply:
Add a comment explaining that this is used for the recursive call, so that similar confusions can be avoided in the future.